README.md

```diff
@@ -151,9 +151,3 @@ with NodeNetworkConfigurationPolicy(
     nncp.add_interface(name="{{ capture.second-nic.interfaces.0.name }}", set_ipv4="{{ capture.first-nic.interfaces.0.ipv4 }}")
     yield nncp
 ```
-
```

Comment:
I don't think we should remove this from the README

Reply:
fine with either way; added back